src/utils/ragHelper.ts

```
@@ -93,44 +94,49 @@ export function calculateSimilarity(query: string, content: string): number {
-    .filter(word => word.length > 1) // 한 글자는 제외
-  
+    .filter((word) => word.length > 1) // 한 글자는 제외
+
   const contentWords = content
     .toLowerCase()
     .replace(/[^\w\s가-힣]/g, ' ')
     .split(/\s+/)
-  
+
   let matchCount = 0
   let totalWords = queryWords.length
```

ESLint

[error] 'totalWords' is never reassigned. Use 'const' instead.

(prefer-const)